packages/compiler-utils/src/analyzers/find-accessed.test.ts

```diff
@@ -0,0 +1,220 @@
+import { parseValue, asSourceFile } from '../astUtils/parser';
+import { expect } from 'chai';
+import { AccesedMembers } from './types';
+import { findAccessedMembers } from './find-accessed';
+import '../dev-utils/global-dev-tools';
+
+
+describe('findAccessedMembers', () => {
+    it('should find defined', () => {
+        const ast = parseValue(`(aParam)=>{
+                const a = 'a';
+                const b = 'b';
+            }
+            `);
+        const expected: AccesedMembers = {
+            accessed: {
+            },
+            defined: {
+                aParam: {},
+                a: {},
+                b: {}
+            },
+            modified: {}
+        };
+        expect(findAccessedMembers(ast)).to.eql(expected);
```

Comment:
how about :
```expect(findAccessedMembers(ast).defined).to.eql( defined: {
                aParam: {},
                a: {},
                b: {}
            });```
? 
I think it makes the test short and its point easier to understand